x-pack/plugins/lens/public/editor_frame_service/editor_frame/suggestion_helpers.ts

```diff
@@ -289,5 +289,9 @@ export function getTopSuggestionForField(
     field,
     mainPalette,
   });
-  return suggestions.find((s) => s.visualizationId === visualization.activeId) || suggestions[0];
+  return (
+    suggestions.find((s) => s.visualizationId === visualization.activeId) ||
+    suggestions.filter((suggestion) => !suggestion.hide)[0] ||
```

Comment:
nit, slightly more performant as you don't have to create a new array only to get its first element: 
```suggestion
    suggestions.find((suggestion) => !suggestion.hide) ||
```